### DISCUSSION TITLE
Calling Peng-Robinson equation properties from Refprop to python

### DISCUSSION THREAD

**Discussion by sannan719** [usnistgov/REFPROP-wrappers#592] [Q&A]
Sir, I have a small query regarding Ref prop 10.0.

I want to call Peng Robinson equation properties (enthalpy, density, temperature, pressure ...) from Ref prop 10.0 to my Python simulation program, for a specific pure fluid. But, I am unable to write its code in Python. Can you please help me out?

Can you please tell me the code to call Peng-Robbinson equation properties instead of default EOS properties?

Thank you very much in advance.

**Comment by ianhbell** [comment]
What information do you have on the species you would like to calculate? What sort of molecules?

**Comment by sannan719** [comment]
I have to calculate cycle thermodynamic points. so the information will be any two parameters (P,T,H,S,Q) resulting the corresponding third parameter as usual. Molecules can be R-23, R-170, R-41 or any pure molecules etc.

**Comment by ianhbell** [comment]
Do you know about this function: https://refprop-docs.readthedocs.io/en/latest/DLL/legacy.html#f/_/PREOSdll ?
